src/api/app/models/bs_request_action_add_role.rb

```diff
@@ -51,6 +51,10 @@ def render_xml_attributes(node)
     node.group(name: group_name, role: role)
   end
 
+  def name
```

Comment:
I didn't managed to create a multi action request for `add_role`. Is this possible? Maybe with some of the other action types its not possible as well and the code is not needed? I'm also happy to check this in an follow up